test/images/nettest/nettest.go

```diff
@@ -210,7 +210,7 @@ func main() {
 	http.HandleFunc("/write", state.serveWrite)
 	http.HandleFunc("/status", state.serveStatus)
 
-	go log.Fatal(http.ListenAndServe(fmt.Sprintf("0.0.0.0:%d", *port), nil))
+	go log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", *port), nil))
 
```

Comment:
Can you explain why `0.0.0.0` was dropped?

Reply:
Dropping it  makes the server start listening in ipv4 and ipv6 ( if available). The ListenAndServe method uses  https://golang.org/src/net/dial.go?s=20989:21043#L677
> For TCP networks, if the host in the address parameter is empty or a literal unspecified IP address, Listen listens on all available unicast and anycast IP addresses of the local system